tests/unit/server/monkey_tests.py

```diff
@@ -2,15 +2,24 @@
 import queue
 import unittest
 
+import gevent.monkey
 import gevent.queue
 
+from baseplate.server.monkey import gevent_is_patched
 from baseplate.server.monkey import patch_stdlib_queues
 
 
 class MonkeyPatchTests(unittest.TestCase):
+    def tearDown(self):
```

Comment:
I moved this cleanup to the teardown because otherwise if this test fails it can interfere with other tests.